src/plugins/intel_gpu/src/kernel_selector/cl_kernels/rope_ref.cl

```diff
@@ -4,6 +4,52 @@
 
 #include "include/fetch_utils.cl"
 
+#ifdef CHATGLM4
```

Comment:
This kernel looks very similar to `CHATGLM` kernel. Can it be reused? By choosing the appropriate gws

Reply:
I merged chatglm4 kernel to chatglm kernel. please check it again.